wow/views.py

```diff
@@ -38,7 +38,6 @@ def clean_addr_dict(addr):
         "bin": str(addr['bin']),
         "lastsaleamount": int_or_none(addr['lastsaleamount']),
         "registrationid": str(addr['registrationid']),
-        "rspercentchange": float_or_none(addr['rspercentchange']),
```

Comment:
Oh ok, it looks like the TypeScript typing changes to this, and the removal of `avrspercent`, are in #414.  Are we removing these b/c the frontend doesn't use them at all?

That's fine by me--I forget if this endpoint is the one used by DAP portal though, do you think it uses these fields?

Reply:
That is correct! And hmmm good point about DAP Portal— they don't use the wow table but they do use our aggregate function. I just pinged Jade to make sure they know that we are removing that field. They are def not using any rs datapoints on DAP Portal from us but you're right that we should flag it for them so they can prepare for it in case there is any validation in place for the API response they get from us.